lib/Cake/Model/Datasource/Database/Mysql.php

```diff
@@ -141,6 +141,13 @@ class Mysql extends DboSource {
  */
 	protected $_charsets = array();
 
+/**
+ * Map of the columns contained in a result.
+ *
+ * @var array
+ */
+	public $map;
```

Comment:
Adding new public properties is a tad more than improving documentation.

Reply:
This is not a new property. See https://api.cakephp.org/2.10/source-class-Mysql.html#255.

Reply:
Okay, but why not declare it in parent class `DboSource` instead of doing it in class for each database?

Reply:
Good point. I was not sure whether other classes that possibly extend DboSource need that.